.darglint

```diff
@@ -0,0 +1,4 @@
+[darglint]
+docstring_style=sphinx
+enable=DAR104
+strictness=long
```

Comment:
can we make this full?

Reply:
I guess we could but wouldn't it be too strict for the initial integration? We already have like a million violations. This would probably need a noqa in every place. Do you think this is sustainable at this point?

Reply:
But yeah, we could totally do
```suggestion
strictness=full
```
if that's not a concern.

Reply:
since we have all the per-file exceptions this should be fine right, and cover new files?

Reply:
I'm actually not fully sure but I'm not strongly opposed to accepting your suggestion either. Thought I'd mention this just in case so that it's explicit. The config is mostly copied from other projects that sometimes have different considerations.